app/Domain/Integrations/Controllers/IntegrationController.php

```diff
@@ -263,12 +274,15 @@ public function activateWithOrganization(string $id, CreateOrganizationRequest $
 
         $this->integrationRepository->activateWithOrganization(Uuid::fromString($id), $organization->id);
 
-        return Redirect::route(
-            TranslatedRoute::getTranslatedRouteName($request, 'integrations.show'),
-            [
-                'id' => $id,
-            ]
-        );
+        return Redirect::back();
+    }
+
+    public function distributeAuth0Clients(string $id): RedirectResponse
```

Comment:
I don't think the front end needs to handle the distribution of a key. A key becoming distributed is a "side-effect" of activating an integration. When an integration is activated the production key gets set to distributed.